### PR TITLE
Topic/genotype marker loading

### PR DESCRIPTION
**Pull request by isaak**
Description 
-----------------------------------------------------
-- makes PHG data loading from the command line and downloading from the Search Wizard and other interfaces  possible

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [ ] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

**Comment by titima15** [comment]
@isaak just checked, you are right, now we use only vcf_snp_genotyping cvterm. The old snp genotyping cvterm was updated to this new term.

**Review by titima15 — COMMENTED** [review]
For your loading script, the "snp genotyping" cvterm is now obsoleted. I haven't checked other part, but they all should be using "vcf_snp_genotyping" cvterm

**Comment by isaak** [comment]
Thanks, Titima. Removed usage and references to the 'snp genotyping' cvterm. 2e063f1

File: lib/SGN/Controller/AJAX/LabelDesigner.pm
Comment (titima15) [inline lib/SGN/Controller/AJAX/LabelDesigner.pm:1]: @isaak Please check this file. Looks like you accidentally deleted the codes

File: mason/breeders_toolbox/genotyping_trials/create_genotyping_trial_dialogs.mas
Comment (titima15) [inline mason/breeders_toolbox/genotyping_trials/create_genotyping_trial_dialogs.mas:1]: Please check this file as well, looks like you accidentally deleted codes from master

Comment (isaak) [reply mason/breeders_toolbox/genotyping_trials/create_genotyping_trial_dialogs.mas:1]: eef7673

File: lib/CXGN/Genotype/Search.pm [hunk 1]
```diff
@@ -2271,24 +2267,20 @@ sub get_pcr_genotype_info {
 
 }
 
-sub get_protocol_id_from_genotype_id {
```
Comment (titima15) [inline lib/CXGN/Genotype/Search.pm:2274]: You modified this function including function name, but the "get_protocol_id_from_genotype_id" is being used in line no. 1621

**Review by titima15 — APPROVED** [review]
👍

**Review by lukasmueller — APPROVED** [review]
:-)

**Comment by lukasmueller** [comment]
Oops, this branch has a conflict with master. @Isaak, could you fix the conflict, then I will merge asap.

**Pull request merged**